46.md

```diff
@@ -0,0 +1,151 @@
+NIP-46
+======
+
+Nostr Connect
+------------------------
+
+`draft` `optional` `author:tiero` `author:giowe` `author:vforvalerio87`
+
+## Rationale
+
+Having to enter your Nostr private key on each website or random app _sucks_.  
+Entering your private key on multiple websites or apps increases the risk of it being exposed to hackers or malicious actors. Additionally, there is a risk of falling victim to a phishing attack if you enter your private key on a fake website or app that is designed to look like a legitimate service.
+
+
+## `TL;DR`
+
+
+**App** (*typically a web app, it generates a random ephemeral keypair*) and **Signer** (*typically a mobile app, it holds the private key of the user that represents his Nostr account*) send to each other `kind:4` encrypted DMs, using a relay of choice. 
```

Comment:
Establish terms. TLDR sounds like optional reading and shouldn't be required to define terms.

```suggestion
## Terms

* **App**: Nostr app on any platform that *requires* to act on behalf of a nostr account.
* **Signer**: Nostr app that holds the private key of a nostr account and *can sign* on its behalf.


## `TL;DR`


**App** and **Signer** send [NIP-04](/04.md) encrypted messages to each other, using a relay of choice. 
```